src/test/java/nl/dtl/fairmetadata4j/utils/MetadataUtilsTest.java

```diff
@@ -142,9 +142,25 @@ public void testParseNoTitleStatement() throws Exception {
         fail("This test is execpeted to throw an error");
     }    
     
-    public void testNoTitleWithFlag() {
-        
-    }
+    /**
+     * Test missing rdf statement but the mandatory check is disabled,
+     * so this test is excepted to pass
+     * 
+     * @throws Exception 
+     */
+    @Test
+    public void testParseNoTitleStatementWithFlag() throws Exception {
+        System.out.println("Test : Missing title statement");
```

Comment:
Well 'want' might be too much, but if you use a logger (which will only add 3 lines: one for the logger declaration and 2 for the imports) you can at least control the output through the logging configuration. Right now the only control you have is to comment out the line. What do you think?